Remove babelfishpg_tsql.enable_ownership_structure.

pratikzode opened the pull request:
Previously, The GUC babelfishpg_tsql.enable_ownership_structure no longer serves a purpose as Babelfish cannot run without the ownership structure. Now have removed ownership structure. 

Task: BABEL-3154
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
“Remove babelfishpg_tsql.enable_ownership_structure.
Previously, The GUC babelfishpg_tsql.enable_ownership_structure no longer serves a purpose as Babelfish cannot run without the ownership structure. Now have removed ownership structure.
Task: BABEL-3154
Signed-off-by:Pratik Zode<pzode@amazon.com>“

Signed-off-by: pratikzode <pratik.zode@gmail.com>

xhfanhe commented on contrib/babelfishpg_tsql/src/guc.c:
```diff
@@ -562,7 +562,7 @@ define_custom_variables(void)
 				 gettext_noop("Enable Babelfish Ownership Structure"),
```
Shouldn't we remove this GUC completely?

xhfanhe commented on contrib/babelfishpg_tsql/src/guc.c:
```diff
@@ -1472,8 +1472,4 @@ get_migration_mode(void)
 	return (MigrationMode) migration_mode;
 }
 
-bool
```
Need to remove this function's declaration in guc.h as well

xhfanhe commented:
We also need to modify ownership.sql
```
./contrib/babelfishpg_tsql/sql/ownership.sql:197:       EXECUTE format('ALTER DATABASE %s SET babelfishpg_tsql.enable_ownership_structure = true', CURRENT_DATABASE());
./contrib/babelfishpg_tsql/sql/ownership.sql:198:       EXECUTE 'SET babelfishpg_tsql.enable_ownership_structure = true';
./contrib/babelfishpg_tsql/sql/ownership.sql:212:       EXECUTE format('ALTER DATABASE %s SET babelfishpg_tsql.enable_ownership_structure = false', CURRENT_DATABASE());
```

macwoody commented:
> We also need to modify ownership.sql
> 
> ```
> ./contrib/babelfishpg_tsql/sql/ownership.sql:197:       EXECUTE format('ALTER DATABASE %s SET babelfishpg_tsql.enable_ownership_structure = true', CURRENT_DATABASE());
> ./contrib/babelfishpg_tsql/sql/ownership.sql:198:       EXECUTE 'SET babelfishpg_tsql.enable_ownership_structure = true';
> ./contrib/babelfishpg_tsql/sql/ownership.sql:212:       EXECUTE format('ALTER DATABASE %s SET babelfishpg_tsql.enable_ownership_structure = false', CURRENT_DATABASE());
> ```

rdsutils.c actually relies on enable_ownership_structure as an indication of BBF being installed. So I think we'd have to keep this line for a while

pratikzode committed:
Update on Remove babelfishpg_tsql.enable_ownership_structure
Had removed the Enable Babelfish Ownership Structure from gun.c and also removed this function's declaration in guc.h as well

Task: BABEL-3154
Signed-off-by: pratikzode <pratik.zode@gmail.com>

macwoody commented on contrib/babelfishpg_tsql/src/guc.c:
```diff
@@ -600,7 +591,7 @@ define_custom_variables(void)
 				   NULL,
 				   &pltsql_server_collation_name,
-				   "sql_latin1_general_cp1_ci_as",
+				   "sql_latin1_general_cp1_ci_as",	
```
I think we don't need this line of change

pratikzode committed:
Update on guc.c file

Task: BABEL-3154
Signed-off-by: pratikzode <pratik.zode@gmail.com>
